src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -130,3 +135,11 @@ def load_command_table(self, _):
         g.command('list', 'list_authorization_rules')
         g.show_command('show', 'get_authorization_rule')
         g.command('keys list', 'list_keys')
+
+# MigrationConfigs Region
+    with self.command_group('servicebus migration', sb_migration_util, client_factory=migration_mgmt_client_factory) as g:
+        g.command('start', 'create_and_start_migration')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.command('complete', 'complete_migration', exception_handler=empty_on_404)
+        g.command('stop', 'revert')
```

Comment:
If the actual API endpoint is called "revert", that has a very different connotation than "stop".  Which is more correct?

Reply:
abort would be more correct.